_includes/nav.html

```diff
@@ -22,7 +22,8 @@
               <ul class="dropdown-menu dropdown-menu-dark" aria-labelledby="header-nav-whatWeDo-dropdown">
                 <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'services') %} active" aria-current="page"{% else %}"{% endif %} href="{{ '/services/' | relative_url }}">Services</a></li>
                 <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'equipment') %} active" aria-current="page"{% else %}"{% endif %}  href="{{ '/equipment/' | relative_url }}">Equipment</a></li>
-                <li><hr class="dropdown-divider"></li>
+                <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'FAQ') %} active" aria-current="page"{% else %}"{% endif %}  href="{{ '/FAQ/' | relative_url }}">FAQ</a></li>
```

Comment:
`page.nav_page_specific == 'FAQ` should match the `nav_page_specific` in `faq.md`. So it should change to `faq` (all lowercase).